src/EditorFeatures/Test2/IntelliSense/VisualBasicCompletionCommandHandlerTests_XmlDoc.vb

```diff
@@ -354,7 +354,7 @@ End Class
 
                 state.SendInvokeCompletionList()
                 Await state.AssertCompletionSession()
-                state.SendSelectCompletionItem("!--")
+                ' state.SendSelectCompletionItem("!--")
```

Comment:
Actually, in this case the test is skipped because the method is not yet provided in the editor. Therefore, we cannot verify it. However, I agree it should be better explained. I uncommented method and added a "NotImplemented" implementation of it.

---
In reply to: [208425806](https://github.com/dotnet/roslyn/pull/29016#discussion_r208425806) [](ancestors = 208425806)